modules/services/window-managers/hyprland.nix

```diff
@@ -237,7 +237,8 @@ in {
 
       onChange = lib.mkIf (cfg.package != null) ''
         ( # Execute in subshell so we don't poision environment with vars
```

Comment:
This comment can also be removed, since we're no longer setting `$HIS` a bunch of times.

Reply:
Nice! Addressed.